core/modules/admin_bar/admin_bar.inc

```diff
@@ -766,22 +774,39 @@ function admin_bar_links_alert() {
 function admin_bar_theme_settings() {
   $config = config('admin_bar.settings');
   $form['#config'] = 'admin_bar.settings';
-  $form['margin_top'] = array(
+
+  $form['settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Settings'),
+  );
+  $form['settings']['margin_top'] = array(
     '#type' => 'checkbox',
     '#title' => t('Adjust top margin'),
     '#default_value' => $config->get('margin_top'),
     '#description' => t('Shifts the entire site content down to make room for the administration bar. If disabled, absolute- or fixed-positioned page elements may be covered by the administration bar.'),
   );
-  $form['position_fixed'] = array(
+  $form['settings']['position_fixed'] = array(
     '#type' => 'checkbox',
     '#title' => t('Keep bar at top of page'),
     '#default_value' => $config->get('position_fixed'),
     '#description' => t('Displays the administration bar always at the top of the browser viewport, even when scrolling the page.'),
   );
+  $form['settings']['local_actions_bottom'] = array(
+    '#type' => 'checkbox',
```

Comment:
I think this might work/look better as a set of radio buttons: top & bottom...

Reply:
Hmm 🤔 ...I don't object to that, but the same can be said for almost all settings in this form that are currently checkboxes.

Anyway, I think I prefer that too, because it would additionally reduce the help text. I'll update it shortly.